lua/idris2/config.lua

```diff
@@ -10,6 +10,7 @@ local defaults = {
   },
   -- opts for nvim-lspconfig
   server = {},
+  post_hooks = {},
```

Comment:
I think `code_action_post_hooks` or `interactive_edits_post_hooks` would be a more explicit name.